assets/css/stripeStyle.css

```diff
@@ -252,6 +253,10 @@ table tr {
 	color: #304ffe;
 }
 
+.toggle {
```

Comment:
rename this class in is-hidden

Reply:
The toggle should be visible on mobile, the style of that starts at line 406.
I only hide it on desktop breackpoint. Should I change it anyway?